src/imagelist.c

```diff
@@ -299,12 +344,14 @@ void image_list_destroy(void)
 
 void image_list_add(const char* source)
 {
-    struct stat st;
+    struct stat st, dir_st;
+    memset(&st,     0, sizeof(struct stat));
```

Comment:
Please, use clang-format to format the code. The root dir contains the file with rules for that.